src/client/src/orangehrmPerformancePlugin/pages/KpiSearch.vue

```diff
@@ -91,20 +97,7 @@ import {APIService} from '@/core/util/services/api.service';
 import usePaginate from '@/core/util/composable/usePaginate';
 import useSort from '@/core/util/composable/useSort';
 import JobtitleDropdown from '@/orangehrmPimPlugin/components/JobtitleDropdown.vue';
-
-const kpiNormalizer = data => {
-  return data.map(item => {
-    return {
-      id: item.id,
-      title: item.title,
-      jobTitleName: item.jobTitle.name,
-      jobTitleId: item.jobTitle.id,
-      minRating: item.minRating,
-      maxRating: item.maxRating,
-      isDefault: item.isDefault ? 'Yes' : '',
-    };
-  });
-};
+import usei18n from '@/core/util/composable/usei18n';
```

Comment:
@Super-Chama 
`usei18n` or `useI18N` :thinking: 

https://vue-i18n.intlify.dev/guide/advanced/composition.html#explicit-with-usei18n

Reply:
hmm.. I used former since [i18n](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/i18n) is like the standard term. Let's refactor after merge ?